src/typeahead/typeahead.js

```diff
@@ -399,13 +399,34 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
     };
   }])
 
-  .filter('typeaheadHighlight', function() {
+  .filter('typeaheadHighlight', ['$sce', '$injector', '$log', function($sce, $injector, $log) {
+
+    var $sanitize;
+    try {
+      $sanitize = $injector.get('$sanitize'); //tries to inject the sanitize service
```

Comment:
Since you're just checking for the existence, you can do `$injector.has('$sanitize')`. Also, probably can call this `hasSanitize` instead.